components/supervisor/pkg/supervisor/supervisor.go

```diff
@@ -328,21 +328,23 @@ func Run(options ...RunOption) {
 
 	if !cfg.isHeadless() {
 		go func() {
-			<-cstate.ContentReady()
-
-			start := time.Now()
-			defer func() {
-				log.Debugf("unshallow of local repository took %v", time.Since(start))
-			}()
-
-			cmd := runAsGitpodUser(exec.Command("git", "fetch", "--unshallow", "--tags"))
-			cmd.Env = childProcEnvvars
-			cmd.Dir = cfg.RepoRoot
-			cmd.Stdout = os.Stdout
-			cmd.Stderr = os.Stderr
-			err := cmd.Run()
-			if err != nil {
-				log.WithError(err).Error("git fetch error")
+			for _, repoRoot := range strings.Split(cfg.RepoRoots, ",") {
```

Comment:
Is it backward compatible? If supervisor goes first?

Reply:
No, is that a deployment mode, that new supervisor gets initialized with old ws-manager? It won't initialize any repositories if the new env variable is not set. Also I don't know how supervisor behaves if the env variable doesn't exist.